src/main/resources/static/templates/CrossSiteRequestForgeryVulnerability/LEVEL_1/CrossSiteRequestForgeryVulnerability.css

```
@@ -0,0 +1,120 @@
+#CrossSiteRequestForgeryVulnerability .csrf-btn-secondary:hover {
+  background-color: #616161;
+}
+
+#CrossSiteRequestForgeryVulnerability .csrf-response {
+  display: none;
+  margin-top: 12px;
+  padding: 10px;
+  border-left: 3px solid #4caf50;
+  background-color: #f9f9f9;
+  font-size: 13px;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)